frontend/src/pages/skills.jsx

```diff
@@ -82,3 +82,123 @@ function SkillsSelector() {
 }
 
 export default SkillsSelector;
+
```

Comment:
Do these comments need to be here? If it's not going to be used it's best to delete it